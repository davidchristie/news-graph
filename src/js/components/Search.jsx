import React from 'react'

export default class Search extends React.Component {
  render() {
    return (
      <nav className="navbar">
        <form className="form-inline">
          <input className="form-control mr-sm-2" type="text" placeholder="Search" />
          <button className="btn btn-outline-success my-2 my-sm-0" type="submit">Search</button>
        </form>
      </nav>
    )
  }
}
