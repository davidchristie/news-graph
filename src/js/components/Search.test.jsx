/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import Search from './Search'

test('Search renders correctly', () => {
  const tree = renderer.create(
    <Search />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
