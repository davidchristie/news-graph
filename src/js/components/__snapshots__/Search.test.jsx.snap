exports[`test Search renders correctly 1`] = `
<form
  className="form-inline my-2 my-md-0">
  <input
    className="form-control mr-sm-2"
    placeholder="Search"
    type="text" />
  <button
    className="btn btn-outline-success my-2 my-sm-0"
    type="submit">
    Search
  </button>
</form>
`;
